src/stepFunctions/commands/visualizeStateMachine/aslVisualization.ts

```diff
@@ -62,7 +62,7 @@ export class AslVisualization {
 
     public async sendUpdateMessage(updatedTextDocument: vscode.TextDocument) {
         const logger: Logger = getLogger()
-        const isYaml = updatedTextDocument.languageId === 'yaml'
+        const isYaml = updatedTextDocument.languageId === 'yasl'
```

Comment:
Should this also accept YAML files? What happens to users who are currently visualizing with `.yaml` files?